docs/cmdline/create.md

```diff
@@ -0,0 +1,60 @@
+---
+title: terramate create - Command
+description: With the terramate create command you can create a new stack in the current project.
+
+prev:
+  text: 'Stacks'
+  link: '/stacks/'
+
+next:
+  text: 'Sharing Data'
+  link: '/data-sharing/'
+---
+
+# Create
+
+The `create` command creates a new stack. This command accepts a path that will
+be used to create the stack in. The `create` command will create intermediate
+directories as required.
+
+## Usage
+
+Usage:
+
+`terramate create [path] [options]`
+
+## Examples
+
+Create a new stack:
+
+```bash
+terramate create path/to/stack`
+```
+
+Complete Example:
+
+```bash
+terramate create path/to/stack \
+    --name foo-stack \
+    --description "This is an example" \
+    --tags "app" \
+    --tags "prod" \
+    --after ../../after-another-stack,../../after-this-stack \
+    --before ../../before-this-stack \
+    --before ../../before-another-stack \
+    --ignore-existing \
+    --no-generate
+```
+
+## Options
+
+- `--id=STRING` ID of the stack. Defaults to UUID.
+- `--name=STRING` Name of the stack. Defaults to stack dir base name.
+- `--description=STRING` Description of the stack. Defaults to the stack name.
+- `--tags=STRING` Adds tags to the stack. `--tags A --tags B` is the same as `--tags A,B`.
+- `--import=STRING`  Add import block for the given path on the stack.
+- `--after=STRING` Add a stack as [after](../orchestration/index.md#explicit-order-of-execution). `--after ../after-this-stack --after after-another-stack` is the same as `--after ../after-this-stack,../after-another-stack`
+- `--before=STRING` Add a stack as [before](../orchestration/index.md#explicit-order-of-execution). `--before ../before-this-stack --before before-another-stack` is the same as `--before ../before-this-stack,../before-another-stack`
```

Comment:
```suggestion
- `--tags=LIST` Adds tags to the stack. `--tags A --tags B` is the same as `--tags A,B`.
- `--import=LIST`  Add import block for the given path on the stack.
- `--after=LIST` Add a stack as [after](../orchestration/index.md#explicit-order-of-execution). `--after ../after-this-stack --after after-another-stack` is the same as `--after ../after-this-stack,../after-another-stack`
- `--before=LIST` Add a stack as [before](../orchestration/index.md#explicit-order-of-execution). `--before ../before-this-stack --before before-another-stack` is the same as `--before ../before-this-stack,../before-another-stack`
```

Reply:
In that case this is unfortunately also documented wrong in the output of `--help`

Reply:
@mariux but this could be `LIST` or `STRING`